AnagramSolver.WebApp/Controllers/AnagramController.cs

```diff
@@ -34,7 +36,16 @@ public IActionResult Details(string id)
             var vocabularyByModel = new HashSet<AnagramViewModel>();
             var allWords = _wordRepository.GetAllWords();
             var _anagramSolver = new BusinessLogic.Classes.AnagramSolver(allWords);
-            var anagramsById = _anagramSolver.GetAnagrams(id);
+            List<string> anagramsById;
+            if (_cachedAnagrams.GetCachedAnagram(id) != null)
+            {
+                anagramsById = _cachedAnagrams.GetCachedAnagram(id);
```

Comment:
GetCachedAnagram(id); called twice. It's enough to be called once.

Reply:
done